tests/v3/connection_state_test.c

```diff
@@ -244,16 +238,22 @@ static int s_setup_mqtt_server_fn(struct aws_allocator *allocator, void *ctx) {
     ASSERT_SUCCESS(aws_condition_variable_init(&state_test_data->cvar));
     ASSERT_SUCCESS(aws_mutex_init(&state_test_data->lock));
 
-    /* Sleep for one millisecond to insure timestamp doesn't repeat over tests */
-    aws_thread_current_sleep(ONE_MILLISECOND);
-    uint64_t timestamp = 0;
-    ASSERT_SUCCESS(aws_sys_clock_get_ticks(&timestamp));
+    struct aws_byte_buf endpoint_buf =
+        aws_byte_buf_from_empty_array(state_test_data->endpoint.address, sizeof(state_test_data->endpoint.address));
+#ifdef _WIN32
+    AWS_FATAL_ASSERT(
+        aws_byte_buf_write_from_whole_cursor(&endpoint_buf, aws_byte_cursor_from_c_str("\\\\.\\pipe\\testsock")));
+#else
+    AWS_FATAL_ASSERT(aws_byte_buf_write_from_whole_cursor(&endpoint_buf, aws_byte_cursor_from_c_str("testsock")));
```

Comment:
I'm confused on what this is doing. Isn't it just going to be overridden in line 249, when you call `aws_uuid_to_str()`? Or does it append it to the end of the buffer?

Reply:
Never mind, I just checked and it appends it to the buffer. 😅